layouts/partials/hooks/head-end.html

```diff
@@ -0,0 +1,4 @@
+{{/* Include link to blog canonical URL, if "canonical_url" field exists in front matter. */}}
+{{ if .Params.canonical_url }}
+<link rel="canonical" href="{{ .Params.canonical_url }}">
+{{ end }}
```

Comment:
Thanks for this @jack-berg, but I'd rather that we address this via a separate PR (possibly with this approach or another).

Reply:
Seems reasonable, split out to #1649.